lib/csv_importer/csv_reader.rb

```diff
@@ -45,6 +45,7 @@ def read_content
 
     def sanitize_content(csv_content)
       internal_encoding = encoding.split(':').last
+      csv_content = csv_content.force_encoding('UTF-8')
```

Comment:
This will depend on the DB configuration... it  can be dangerous to use force encoding. Would be good to know what is the encoding for db, .rb files and web server.

However, according to ticket description:

> When importing a CSV file into TapGoods, items that have names that include ("and ') are appearing in the inventory without any of the quotations/ apostrophes. 

I do not see how this change could fix the issue. `"` and `'` exist even in ASCII charset.

Reply:
also, why, instead of editing the library we do not add the force_encoding to the content to CSV? that seems safer for me.